src/logo/Logo.tsx

```diff
@@ -9,7 +9,7 @@ export class Logo extends React.Component<any, any> {
           <g id="logo">
             <path
               d="M20,20 L818,20 C829.045695,20 838,28.954305 838,40 L838,192 C838,203.045695 829.045695,212 818,212 L20,212 C8.954305,212 1.25158157e-14,203.045695 -3.55271368e-15,192 L-3.55271368e-15,40 C-4.90542118e-15,28.954305 8.954305,20 20,20 Z"
-              id="bg" fillOpacity="0.25" fill="#000"/>
+              id="bg" className="asd" fillOpacity="0.25" fill="#000"/>
```

Comment:
why add this class if you dont use it? And if add, then make it meaningfull and use in styles instead of `#bg`

Reply:
left overs